### ISSUE TITLE
Modal focus should to remain in the view

### ISSUE THREAD

**Issue by joevin22** [angular-ui/bootstrap#4289]
When a modal is opened, the focus should to remain in the modal view on press "tab" key for accessibility.

tabindex of modal view should to be in priority.

**Comment by wesleycho** [comment]
Can you explain this more clearly? We currently do attempt to trap the focus inside the modal, but there is another issue filed against that behavior. If that is what you are filing an issue on, I can close this as a duplicate. If not, please elaborate clearly.

**Comment by joevin22** [comment]
Yes, 
When modal view is opened and that we press tabkey many times, focus is behind the modal.
It should to remain on modal dialog.

What is the issue opening ? i did not see it.

**Comment by icfantv** [comment]
per comments from @joevin22, this issue is a duplicate of #738.

**Comment by wesleycho** [comment]
Can you post a simple reproduction based on [Plunker](http://plnkr.co/edit/ypzzUCLwUwnDh9ScipVf?p=preview)? I cannot reproduce this.

**Comment by joevin22** [comment]
Ok, i forgot to upgrade angular-bootstrap to 0.13.3,
bug is fixed it, the issue can to be closed, 
Thx

**Comment by joevin22** [comment]
Ok, i forgot to upgrade angular-bootstrap to 0.13.3,
bug is fixed it, the issue can to be closed, 
Thx

**Comment by joevin22** [comment]
Ok, i forgot to upgrade angular-bootstrap to 0.13.3,
bug is fixed it, the issue can to be closed, 
Thx

**Comment by joevin22** [comment]
Ok, i forgot to upgrade angular-bootstrap to 0.13.3,
bug is fixed it, the issue can to be closed, 
Thx
